packages/onboarding-ui/src/common/ManageWorkspaceFallback.spec.tsx

```diff
@@ -0,0 +1,12 @@
+import ReactDOM from 'react-dom';
+
+import ManageWorkspaceFallback from './ManageWorkspaceFallback';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(
+    <ManageWorkspaceFallback onManageWorkspaceClick={() => undefined} />,
+    div
+  );
+  ReactDOM.unmountComponentAtNode(div);
+});
```

Comment:
shouldn't this be removed too?

Reply:
or is the component being used again elsewhere?

Reply:
We understood that maybe we use this in the future, in this way not is necessary to remove it.